pkg/server/authpolicy/policy.rego

```diff
@@ -32,15 +32,15 @@ default allow = false
 
 
 # Admin allow check
-allow_if_admin = true { 
```

Comment:
Is this new syntax backwards-compatible or do we need to maintain two separate Rego policies for v0 Rego and v1 Rego?

Reply:
For this particular file I don't think we need to care, it's only used if another policy is not specified. It uses an embed file and we can specify the correct version for it.

As for backwards-compatibility, I don't think there is any. RegoV0 seems to load this file just file, but it probably misinterprets it because our auth tests are failing if I do that.